go/pkg/credshelper/credshelper.go

```diff
@@ -209,53 +240,50 @@ func runCredsHelperCmd(credsHelperCmd *reusableCmd) (*oauth2.Token, time.Time, e
 		log.Errorf("Credentials helper warnings and errors: %v", stderr.String())
 	}
 	if err != nil {
-		return nil, time.Time{}, err
+		return credshelperOutput{}, err
 	}
-	token, expiry, refreshExpiry, err := parseTokenExpiryFromOutput(out)
-	return &oauth2.Token{
-		AccessToken: token,
-		Expiry:      expiry,
-	}, refreshExpiry, err
+	return parseTokenExpiryFromOutput(out)
 }
 
-// CredsHelperOut is the struct to record the json output from the credshelper.
-type CredsHelperOut struct {
-	Token         string `json:"token"`
-	Expiry        string `json:"expiry"`
-	RefreshExpiry string `json:"refresh_expiry"`
+// JSONOut is the struct to record the json output from the credshelper.
+type JSONOut struct {
+	Token         string            `json:"token"`
```

Comment:
I think go lint will complain if these fields aren't documented? I'm not sure (but it will only really show up as a problem when we import to Google3, so perhaps just better to document the fields now).

Reply:
I don't think it does, since it didn't complain when I imported the first credshelper package change which already had this.

Reply:
Sg then.